connectors/cassandra/schema.go

```diff
@@ -34,11 +34,13 @@ import (
 // CreateScope creates a keyspace
 func (c *Connector) CreateScope(ctx context.Context, scope string) error {
 	// drop the old scope, ignoring errors
-	_ = c.DropScope(ctx, scope)
-
+	err := c.DropScope(ctx, scope)
+	if err != nil {
+		fmt.Errorf("drop scope failed: %v", err)
```

Comment:
This isn't really an error, so downgrade this to at least a warning, or perhaps just info.

Reply:
yeah, I was trying to just warn it. Cannot find how we do warning here since there is no logger configured in this repo. 

`Info` is fine too.

Reply:
Changed to just print out the error info.